cpp/ql/src/Security/CWE/CWE-190/ArithmeticUncontrolled.ql

```diff
@@ -59,10 +59,15 @@ predicate boundedBitwiseAnd(Expr e, Expr andExpr, Expr operand1, Expr operand2)
 }
 
 /**
- * Holds if `fc` is a part of the left operand of a binary operation that greatly reduces the range
- * of possible values.
+ * Holds if `e` is an operand of an operation that greatly reduces the range of possible values.
  */
 predicate bounded(Expr e) {
+  (
+    e instanceof UnaryArithmeticOperation or
+    e instanceof BinaryArithmeticOperation
```

Comment:
What about `AssignArithmeticOperation`?

Reply:
Good point. I've added this one in bb6a2d3845a43b95c8dc23195857977e06087448.